draft-ietf-snac-simple.xml

```diff
@@ -513,12 +521,12 @@
 	    routing fabric, so even though packets might be able to be forwarded to the intended destination, there would be no
 	    return path. So when the only prefix that is available is the one provided by the stub router, cloud services will
 	    not be reachable via IPv6, and infrastructure-provided NAT64 will not work. Therefore, when the stub router is able to
-	    successfully acquire a prefix using DHCPv6 PD, it MUST use DHCPv6 PD rather than its own self-generated ULA prefix.
+	    successfully acquire a prefix using DHCPv6 PD, it MUST use DHCPv6 PD rather than its own self-generated ULA Site Prefix.
```

Comment:
We could say here "rather than the ULA Link prefix it allocated for the stub network out of its ULA Site Prefix."

Reply:
Updated.